projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/customer-ticketing.ts

```diff
@@ -149,3 +155,105 @@ export function verifyTicketDoesNotExist(ticketDetails: TestTicketDetails) {
 export function visitApparelUKTicketListingPage(){
   visitPage('apparel-uk-spa/en/GBP/my-account/support-tickets', 'apparelTicketListingPage');
 }
+
+export function verifyTicketListingTableContent(): number{
+  let numberOfTickets = 0;
+  cy.get('cx-customer-ticketing-list').then( ticketListingElement => {
+    if(ticketListingElement.find('tbody').length > 0) {
+      const headerRow = cy.get('cx-customer-ticketing-list').get('tbody').get('tr').eq(COLUMN_HEADER_TICKET_LIST).get('td');
+      const allRowColumns = headerRow.get('td');
+      allRowColumns.eq(ID_COLUMN).should('contain', 'ID');
+      allRowColumns.eq(SUBJECT_COLUMN).should('contain', 'Subject');
+      allRowColumns.eq(CATEGORY_COLUMN).should('contain', 'Category');
+      allRowColumns.eq(CREATED_ON_COLUMN).should('contain', 'Created On');
+      allRowColumns.eq(CHANGED_ON_COLUMN).should('contain', 'Changed On');
+      allRowColumns.eq(STATUS_COLUMN).should('contain', 'Status');
+      numberOfTickets = ticketListingElement.find('tbody').length;
+    }
+    else {
+      cy.get('cx-customer-ticketing-list').find('h3').contains("You don't have any request");
+    }
+  });
+
+  return numberOfTickets;
+}
+
+export function getNumberOfTicket(): number {
+  let numberOfTikcets = 0;
+  cy.get('cx-customer-ticketing-list').then(ticketListingElement => {
+    numberOfTikcets = ticketListingElement.find('tbody').length;
+  });
+  return numberOfTikcets;
+}
+
+export function createTicket(ticketDetails: TestTicketDetails){
+  openCreateTicketPopup();
+  fillTicketDetails(ticketDetails);
+  clickSubmit();
+  verifyRequestCompleted();
+}
+
+export function createNumberOfTickets(numberOfTicket: number, ticketDetails: TestTicketDetails){
+  for (let i = 0; i < numberOfTicket; i++)
+  {;
+    createTicket(ticketDetails);
+  }
+}
+
+export function shouldHaveNumberOfTicketsListed(expectedNumberOfTickets: number) {
+  cy.get('cx-customer-ticketing-list').get('tbody').should('have.length', expectedNumberOfTickets);
+}
+
+export function openLastCreatedTicket() {
+  const row = cy.get('cx-customer-ticketing-list').get('tbody').get('tr').eq(FIRST_ROW_TICKET_LIST);
+  row.click();
+}
+
+export function clickCloseRequestButton() {
+  cy.get('cx-customer-ticketing-close').click();
+}
+
+export function typeCloseTicketRequestMessage(message = 'Thank you.') {
+  cy.get('textarea').last().type(message);
+
+}
+
+export function closeTicketRequest(message = 'Thank you.') {
+  const SUBMIT_BUTTON_INDEX = 2;
+  clickCloseRequestButton();
+  cy.get('form').within(() => {
+    typeCloseTicketRequestMessage(message);
+    cy.get('button').eq(SUBMIT_BUTTON_INDEX).click();
+  })
+}
+
+export function verifyClosedTicketIsStillInTicketListing() {
```

Comment:
Can we make it more reusable?
verifyTicketStatusInTicketListing(ticketPostionInList / ticketId,  Status)

So that this function can be used by the re-open journey